content/en/docs/concepts/overview/working-with-objects/conditions.md

```diff
@@ -0,0 +1,102 @@
+
+## Purpose and Usage of Conditions
+
+Conditions being an integral part of Kubernetes play very important role by:
+
+* **Status Information**: Conditions provide an in-depth understanding of the existing cluster state or resources. They help the user or a machine understand what’s going on.
+
+* **Communication**: Conditions act as a form of communication inside the cluster. It helps various components like the kubelet, the scheduler talk to each other or custom controllers to send and receive messages.
+
+* **Activity Engine**: They help custom controllers to perform actions. For given conditions, controllers can make HTTP requests, call to cloud API, etc. Once a pod is unschedulable, a controller might create more resources, Websites, etc.
+
+* **Events**:
+
+* They are used to record a single one-time event.
+
+* **Examples**: Kubernetes uses events to report changes during the control plane’s request processing. It uses events for purely informational purposes, trying to help understand what’s going on in the cluster.
+
+Examples where Conditions are used:
+
+* `FrequentSyncError`: Indicates the resource Seeding controller is struggling to create.
+
+* `ComponentsInstalled`: Indicates that a desired state of components has been installed.
+
+* `Repairing`: Indicates that the controller has started repairing things.
+
+* `Mounted`: Indicates that a controller from the Cloud integration is successfully mounted on the node.
+
+* **Conditions**:
+
+* They provide an engineering statement regarding the running process.
+
+* **Defining Conditions**: When we design conditions, multiple things have to be cared upon.
+
+* **Polarity**: any unhealthy state should be `True`.
+
+* **Extensibility**
+
+* **Meaning**: once a condition is defined, it should have the same meaning. This is because of backward compatibility. Because of this, only a single condition indicating an abnormal state must be defined.
+
+It should be noticed that conditions are observations and not state machines. They are free-formers and don’t have states.
+
+* **Code examples**:
+
+Some not-so-well-known example of how conditions are used in various Kubernetes projects are:
+
+* [Cert-Manager](https://cert-manager.io/docs/concepts/issuer/): The `Ready` condition is used for issuers.
+
+* OpenShift defines custom conditions, like `Available`, `Progressing`, `Degraded`, `Upgradable`, etc.
+
+* [Hyperconverged Cluster Operator](https://github.com/openshift/hypershift): The conditions are used to represent the state of component custom resources.
+
+* Gardene uses two conditions: `Valid` and `Installed` for the controller installation.
+
+## Further Reading
+
+If you want to learn more about conditions and when to use them in Kubernetes, you can refer to:
+
```

Comment:
Ideally, also link to [Pod readiness gates](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-readiness-gate)